code/modules/client/preference/loadout/loadout_uniform.dm

```diff
@@ -61,17 +61,17 @@
 /datum/gear/uniform/skirt/job/viro
 	display_name = "skirt, virologist"
 	path = /obj/item/clothing/under/rank/virologist/skirt
-	allowed_roles = list("Chief Medical Officer","Medical Doctor")
+	allowed_roles = list("Chief Medical Officer","Medical Doctor" , "Virologist")
```

Comment:
I'm not sure why they where in the first place; but I guess since it was first approved by that then there's no reason to remove it unless someone really doesn't like them having it.